torch/_inductor/compile_worker/subproc_pool.py

```diff
@@ -273,14 +317,18 @@ def callback(_: Future[Any]) -> None:
         future.add_done_callback(callback)
 
     @staticmethod
-    def do_job(data: bytes) -> bytes:
+    def do_job(pickler: SubprocPickler, data: bytes) -> bytes:
         # do the pickle/unpickle in the sub-subproc
-        job = pickle.loads(data)
+        shape_env = ShapeEnv()
```

Comment:
I have no idea what's happening here :/

Reply:
Oh - that leaked in from a future PR - I'll move it where it's needed.